var/spack/repos/builtin/packages/py-rasterio/package.py

```diff
@@ -14,25 +14,30 @@ class PyRasterio(PythonPackage):
     arrays."""
 
     homepage = "https://github.com/mapbox/rasterio"
-    url      = "https://github.com/mapbox/rasterio/archive/1.0a12.zip"
+    url      = "https://pypi.io/packages/source/r/rasterio/rasterio-1.0.24.tar.gz"
 
-    version('1.0a12', 'e078ca02b3513b65a9be5bb3f528b4da')
+    maintainers = ['adamjstewart']
+    import_modules = ['rasterio', 'rasterio.rio']
 
-    variant('aws', default=False,
-        description='Enable testing with Amazon Web Services')
```

Comment:
Will removing AWS support negatively affect somebody?  (i.e. whoever put it in..)

Reply:
Idk, you tell me (#7156). As far as I can tell, the AWS dependency is only needed for testing purposes, it does not provide additional functionality. The dependency is still there, but is only activated at test time.

Reply:
Well, that's embarassing.  I think I remember... a friend needed it, so I made the package.  I've never used it myself.  And I have no idea why there's a variant that had no effect.